myplugins/better.starts/data/bstartsStarts.txt

```diff
@@ -3,7 +3,7 @@ start "z_trader1"
 	description `Ship: Freighter`
 	description `Special: 165t Cargo`
 	description `You grew up on New China in the Merak system, an overpopulated, dirty and poor world, dreaming of an escape.`
-	description `Your neighbor, a space trader, goes in pension. He offers you to buy his "Freighter". This can be your chance.`
+	description `Your neighbor, who a space trader, is going into pension. He offers you a chance to buy his "Freighter". This can be your chance.`
```

Comment:
```suggestion
	description `Your neighbor, who is a space trader, is going into pension. He offers you a chance to buy his "Freighter". This can be your chance.`
```